src/app/views/templates/templates.html

```diff
@@ -76,14 +87,15 @@ <h3 ng-if="$ctrl.worksheet === 'Workbook'">{{ worksheet }} Worksheet</h3>
           Save
         </button>
       </div>
-      <uib-accordion close-others="false" ng-repeat="(group, attributes) in data.attributes">
-        <div
-          uib-accordion-group
-          class="panel-default md-whiteframe-2dp"
-          heading="{{ group }}"
-          is-open="true"
-        >
-          <div class="checkbox" ng-repeat="attribute in attributes">
+      <uib-accordion close-others="false">
+        <div uib-accordion-group class="panel-default md-whiteframe-2dp" is-open="true">
+          <div ng-if="$ctrl.worksheet === 'Workbook'" uib-accordion-heading>
+            {{ worksheet }} Worksheet Minimum Information Standard Items
+          </div>
+          <div ng-if="$ctrl.worksheet !== 'Workbook'" uib-accordion-heading>
+            Minimum Information Standard Items
+          </div>
```

Comment:
I think this can be simplified to just:

`<span ng-if="$ctrl.worksheet === 'Workbook'">{{ worksheet }} </span>Minimum Information Standard Items`

I don't think we need the "Worksheet" word in there